src/smif/model/sector_model.py

```diff
@@ -239,4 +170,7 @@ def simulate(self, data):
         ``get_results(output_name, model_name=None, modelset_iteration=None,
         decision_iteration=None, timestep=None)``.
 
+        ``data.get_state()`` returns a list of intervention dict for the
```

Comment:
We should document both and clarify the use cases:
- `data.get_state` returns `list[tuple]` where each tuple is `(name, build_year)`
- `data.get_current_interventions` returns `list[dict]` where each dict is an intervention

Perhaps `get_current_interventions` is more useful for a `Model` whereas `get_state` is useful for a `DecisionModule`?

Reply:
Done